fairlearn/metrics/_metrics_engine.py

```diff
@@ -159,14 +158,14 @@ def __init__(self, transformation_function, summary_function, name=None):
 
     def __repr__(self):
         return "make_derived_metric({0}, {1})".format(
-                _function_name(self._transformation_function),
-                _function_name(self._summary_function))
+            _function_name(self._transformation_function),
+            _function_name(self._summary_function))
 
     def __call__(self, y_true, y_pred, *, sensitive_features, **metric_params):
         return self._transformation_function(self._summary_function(
-                y_true, y_pred,
-                sensitive_features=sensitive_features,
-                **metric_params))
+            y_true, y_pred,
+            sensitive_features=sensitive_features,
```

Comment:
This is not unique to this PR, but I feel the argument `sensitive_features` implies a list of feature names (e.g. `['gender', 'race']`) instead of an array indicating group membership. I'm not sure what a better name would be - perhaps `group` or even just `a` or `s` which is often used in fairness lit.

Reply:
I'd say [hopefully] most people who end up using this package are not gonna be FAT-ML scholars if we achieve a broad adoption of the library, and hence `A` would be very confusing (it certainly is to me). `sensitive_features` however, does give the kinda feeling to people that we want them to get.

As in passing an array or just the feature names, I think eventually we'd support both, but supporting feature names means a pipeline which passes along feature names which we don't have right now and is in active development in sklearn. So I'd say we can keep the name and add feature name support later?

Reply:
Another angle that we were looking at was that `sensitive_features` is a keyword argument that should feel syntactically parallel to `sample_weight`.

Reply:
I'll resolve this specific thread for now, but this naming issue is certainly one we might wish to revisit.